testproject/Assets/Tests/Runtime/GlobalObjectIdHash/NetworkObjectGlobalObjectIdHashTests.cs

```diff
@@ -27,9 +24,7 @@ public IEnumerator Setup()
         {
             SceneManager.sceneLoaded += OnSceneLoaded;
 
-            var execAssembly = Assembly.GetExecutingAssembly();
-            var packagePath = PackageInfo.FindForAssembly(execAssembly).assetPath;
-            var scenePath = Path.Combine(packagePath, $"Tests/Runtime/GlobalObjectIdHash/{nameof(NetworkObjectGlobalObjectIdHashTests)}.unity");
+            const string scenePath = "Assets/Tests/Runtime/GlobalObjectIdHash/" + nameof(NetworkObjectGlobalObjectIdHashTests) + ".unity";
```

Comment:
less programmatic but also less dependent — path looks clear and obvious to me, so I think this is fine.

Reply:
Do these paths work on mobile platforms?

Reply:
AFAIK, they should. IIRC, `Assets/xxx` is a project relative path and not directly OS-dependent.